algorithms/scaling/observers.py

```diff
@@ -243,7 +277,7 @@ def make_scaling_html(self, scaling_script):
             with open(html_file, "wb") as f:
                 f.write(html.encode("utf-8", "xmlcharrefreplace"))
         if json_file:
-            logger.info("Writing html report data to: %s", json_file)
+            logger.info("Writing html report data to %s", json_file)
```

Comment:
`html -> json_file` - surely some mistake? 🤔

Reply:
Actually this one is correct, the html report data can be saved to a json if requested.